packages/twenty-server/src/engine/core-modules/event-emitter/types/object-record-restore.event.ts

```diff
@@ -1,8 +1,8 @@
-import { ObjectRecordCreateEvent } from 'src/engine/core-modules/event-emitter/types/object-record-create.event';
+import { ObjectRecordBaseEvent } from 'src/engine/core-modules/event-emitter/types/object-record.base.event';
 
 export class ObjectRecordRestoreEvent<
   T = object,
-> extends ObjectRecordCreateEvent<T> {
+> extends ObjectRecordBaseEvent<T> {
```

Comment:
This was enforced on purpose because from an API perspective I think the RestoreEvent should always have the properties that the create event has, because people don't want to implemented two different hooks with different logics. I think adding it creates a protection and if there's a need to remove it then it will raise the point